modules/sfp_creditcard.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_creditcard
+# Purpose:      Example module to use for new modules.
+#
+# Author:      Krishnasis Mandal <krishnasis@hotmail.com>
+#
+# Created:     21/04/2020
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+
+class sfp_creditcard(SpiderFootPlugin):
+
+    """Credit Card Extractor:Footprint,Investigate,Passive:Content Analysis::Identify Credit Card Numbers in any data"""
+
+    # Default options.
+    opts = {
+         # Options specific to this module
+    }
+
+    # Option descriptions.
+    optdescs = {
+    }
+
+    # Tracking results can be helpful to avoid reporting/processing duplicates
+    results = None
+
+    # Tracking the error state of the module can be useful to detect when a third party
+    # has failed and you don't wish to process any more events.
+    errorState = False
+
+    def setup(self, sfc, userOpts=dict()):
+        self.sf = sfc
+        # Krishnasis - Note : What does self.tempStorage() return ?
+        self.results = self.tempStorage()
+
+        # Clear / reset any other class member variables here
+        # or you risk them persisting between threads.
+
+        for opt in list(userOpts.keys()):
+            self.opts[opt] = userOpts[opt]
+
+    # What events is this module interested in for input
+    def watchedEvents(self):
+        return ["TARGET_WEB_CONTENT"]
```

Comment:
Do we want to limit this to only web content form the target, since it seems unlikely they'd have actual credit card numbers on their website, right? Maybe we should also look at some of the dark web or leak site content as well?